apps/system/utils/utils_cpuload.c

```diff
@@ -159,19 +197,27 @@ static int cpuload_read_proc(FAR struct dirent *entryp, FAR void *arg)
 		printf("Failed to read %s\n", filepath);
 		return ERROR;
 	}
-	printf("\n");
 	return OK;
 }
 
 static void cpuload_print_normal(void)
 {
 	/* Print titles */
+	printf("\n--------------------------------------------------\n");
+	printf("Non-Zero CPU utilization trend (updated every %ds)\n", cpuload_interval);
 	printf("PID | Pri |");
+
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	printf("%5ds | %4ds | %4ds |", CONFIG_SCHED_CPULOAD_TIMECONSTANT_SHORT, CONFIG_SCHED_CPULOAD_TIMECONSTANT_MID, CONFIG_SCHED_CPULOAD_TIMECONSTANT_LONG);
-#else
-	printf("%5ds |", CONFIG_SCHED_CPULOAD_TIMECONSTANT);
+	for (int i = PROC_STAT_CPULOAD_SHORT; i <= PROC_STAT_CPULOAD_LONG; i++) {
```

Comment:
This is not the right way of printing. When multiload is enabled in single core scenario, then we print the load for 3 cases, short, mid and long time duration. When we have multi-core, then for each duration, we need to print load of each cpu and avg load. Please recheck this - the table heading as well as the data being printed. Once this is done, pls paste a sample output of multiload in the commit.

Reply:
This is for single core case :
```
--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
PID | Pri |  CPU0 |  Avg  |  CPU0 |  Avg  |  CPU0 |  Avg  |
--------------------------------------------------
  0 |   0 |  99.9 |  99.9 |  99.8 |  99.8 |  99.8 |  99.8 | Idle Task
 11 | 203 |   0.0 |   0.0 |   0.1 |   0.0 |   0.1 |   0.1 | binary_manager
 14 | 180 |   0.0 |   0.0 |   0.1 |   0.1 |   0.1 |   0.1 | app1
 19 | 100 |   0.2 |   0.2 |   0.1 |   0.1 |   0.1 |   0.1 | CPULoadMonitor
--------------------------------------------------
```

This is for multi core case:
```
--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
PID | Pri |  CPU0 |  CPU1 |  Avg  |  CPU0 |  CPU1 |  Avg  |  CPU0 |  CPU1 |  Avg  |
--------------------------------------------------
  0 |   0 |  99.9 |   0.0 |  49.7 | 100.0 |   0.0 |  49.3 |  99.9 |   0.0 |  49.4 | CPU0 IDLE
  1 |   0 |   0.0 |  99.9 |  50.3 |   0.0 |  99.8 |  50.6 |   0.0 |  99.8 |  50.5 | CPU1 IDLE
 29 | 100 |   0.2 |   0.1 |   0.1 |   0.1 |   0.1 |   0.1 |   0.1 |   0.1 |   0.1 | CPULoadMonitor
--------------------------------------------------
```

Reply:
In my opinion, multiload scenario will not be used often and we dont have to give details per-cpu information in multiload case. So, for multiload case, let us keep the output simple as follows:
```
--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
PID | Pri |  (%d sec) Avg  |   (%d sec)  Avg  |   (%d sec)   Avg  |
--------------------------------------------------
```

Reply:
These are the updated output . By adding (%d sec) Avg , there was slightly more code added and the output look very spacious , so i updated like this . Is it okay?
```
--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
Multi cpuload mode is enabled with time constants 2s, 5s, 10s (used to average out cpuload after every time constants)
PID | Pri |  Avg1 |  Avg2 |  Avg3 |
--------------------------------------------------
  0 |   0 |  99.8 |  99.7 |  99.7 | Idle Task
 11 | 203 |   0.1 |   0.1 |   0.1 | binary_manager
 14 | 180 |   0.1 |   0.1 |   0.1 | app1
 19 | 100 |   0.2 |   0.1 |   0.1 | CPULoadMonitor
--------------------------------------------------


--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
Multi cpuload mode is enabled with time constants 2s, 5s, 10s (used to average out cpuload after every time constants)
PID | Pri |  CPU0 |  CPU1 |  Avg1 |  CPU0 |  CPU1 |  Avg2 |  CPU0 |  CPU1 |  Avg3 |
--------------------------------------------------
  0 |   0 |  99.8 |   0.0 |  47.9 |  99.9 |   0.0 |  49.0 |  99.9 |   0.0 |  49.3 | CPU0 IDLE
  1 |   0 |   0.0 | 100.0 |  52.0 |   0.0 | 100.0 |  51.0 |   0.0 | 100.0 |  50.6 | CPU1 IDLE
 20 | 100 |   0.4 |   0.0 |   0.2 |   0.1 |   0.0 |   0.1 |   0.1 |   0.0 |   0.1 | CPULoadMonitor
--------------------------------------------------


--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
PID | Pri |  CPU0 |  CPU1 |  Avg  |
--------------------------------------------------
  0 |   0 |  99.9 |   0.0 |  48.6 | CPU0 IDLE
  1 |   0 |   0.0 | 100.0 |  51.3 | CPU1 IDLE
 20 | 100 |   0.1 |   0.0 |   0.1 | CPULoadMonitor
--------------------------------------------------

--------------------------------------------------
Non-Zero CPU utilization trend (updated every 5s)
PID | Pri |  Avg  |
--------------------------------------------------
  0 |   0 |  99.9 | Idle Task
 19 | 100 |   0.1 | CPULoadMonitor
--------------------------------------------------
```

Reply:
Looks fine.